src/amo/components/UserProfileEdit/index.js

```diff
@@ -782,21 +782,31 @@ export class UserProfileEditBase extends React.Component<Props, State> {
                     anonymised and no longer editable.`,
                   )}
             </p>
+            {isEditingCurrentUser && (
+              <p>
+                {i18n.gettext(
+                  `When you use this email address to login again to
+                  addons.mozilla.org, you will create a new Firefox Add-ons
+                  profile that is in no way associated with the profile you
+                  deleted.`,
+                )}
+              </p>
+            )}
             <p
               // eslint-disable-next-line react/no-danger
               dangerouslySetInnerHTML={sanitizeHTML(
                 i18n.sprintf(
                   isEditingCurrentUser
                     ? i18n.gettext(
                         `%(strongStart)sNOTE:%(strongEnd)s You cannot delete
-                        your account if you are the %(linkStart)sauthor of any
```

Comment:
I know this is older code, but should these links open in a new tab/window? It seems odd to interrupt the flow of deleting a user profile by navigating away from this modal. It would take multiple steps to get back here after looking at the information at the linked page(s).

Reply:
I thought we could do it but `DOMPurify` apparently removes the `_target` attribute 😞